src/posix/platform/netif.cpp

```diff
@@ -865,6 +868,8 @@ static otError AddIp4Route(const otIp4Cidr &aCidr, uint32_t aPriority)
 {
     return AddRoute(aCidr.mAddress.mFields.m8, aCidr.mLength, aPriority);
 }
+
+static otError DeleteIp4Route(const otIp4Cidr &aCidr) { return DeleteRoute(aCidr.mAddress.mFields.m8, aCidr.mLength); }
```

Comment:
move return to a single line

Reply:
Formatter is not happy with it :-/

Reply:
Used some hack and the result should be better now.